scripts/nsidc/push_release_to_prod.sh

```diff
@@ -7,8 +7,9 @@ PROD_HOSTNAME='qgreenland.apps.int.nsidc.org'
 VAGRANT_SSH_KEY="$HOME/.ssh/id_rsa_vagrant_vsphere"
 
 echo "WARNING: THIS WILL OVERWRITE PRODUCTION LAYERS WITH LOCAL LAYERS AND SYNC ALL OF YOUR RELEASES TO PRODUCTION."
-read -p "PRESS CTRL-C TO CANCEL. PRESS ENTER TO PUSH STRAIGHT TO PROD AND DESTROY EVERYTHING."
+read -rp "PRESS CTRL-C TO CANCEL. PRESS ENTER TO PUSH STRAIGHT TO PROD AND DESTROY EVERYTHING."
 
-rsync -a --progress --verbose \
+sudo rsync --archive --progress --verbose \
     -e "ssh -i ${VAGRANT_SSH_KEY}" \
+    --rsync-path='sudo rsync' \
```

Comment:
On the VM, everything is owned by root. Without both `sudo` on the sending side and `--rsync-path='sudo rsync'` to use sudo on the receiving side, permission errors occur.